MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,131 @@
+not to make mistakes, that would be hard to fix in the future. In order
+to facilitate this, the "final" decision is made in two stages:
+
+  * We definitely want to try something out
+
+  * We think that the attempt was successful
+
+Technically, the former means "the patch is accepted into criu-dev branch",
+while the latter -- "the patch is accepted into the master branch".
+
+## Who decides what?
+
+All decisions are patches (either in form of pull requests, or sent to 
+the mailing list), and maintainers make decisions by merging or rejecting
+them. Review and approval or disagreement can be done by anyone and is
+denoted by adding a respective comment in the pull request. However,
+merging the change into either branch only happens after approvals from
+maintainers.
+
+In order for a patch to be merged into the criu-dev branch at least two
+maintainers should accept it. In order for a patch to be merged into the
+master branch the majority of maintainers should decide that (then prepare
+a pull request, submit it, ...)
+
+Overall the maintainer system works because of mutual respect across the
+maintainers of the project.  The maintainers trust one another to make
+decisions in the best interests of the project.  Sometimes maintainers
+can disagree and this is part of a healthy project to represent the point
+of views of various people. In the case where maintainers cannot find
+agreement on a specific change the role of a Chief Maintainer comes into
+play.
+
+### Chief maintainer
+
+The chief maintainer for the project is responsible for overall architecture
+of the project to maintain conceptual integrity.  Large decisions and
+architecture changes should be reviewed by the chief maintainer.
+
+Also the cheif maintainer has the veto power on any change submitted
+to any branch. Naturally, a change in the criu-dev branch can be reverted
+after a chief maintainer veto, a change in the master branch must be
+carefully reviwer by the chief maintainer and vetoed in advance.
+
+### How are maintainers added (and removed)?
+
+The best maintainers have a vested interest in the project.  Maintainers
+are first and foremost contributors that have shown they are committed to
+the long term success of the project.  Contributors wanting to become
+maintainers are expected to be deeply involved in contributing code,
+patches review, and paying needed attention to the issues in the project.
+Just contributing does not make you a maintainer, it is about building trust
+with the current maintainers of the project and being a person that they can
+depend on and trust to make decisions in the best interest of the project.
+
+When a contributor wants to become a maintainer or nominate someone as a 
+maintainer, one can submit a "nomination" change, which technically is the
+respective modification to the `MAINTAINERS` file. When a maintainer feels
+he's unable to perform the required duties, or someone else wants to draw
```

Comment:
*he or she* is unable

Reply:
they? :-D

Reply:
> they? :-D

When a maintainer feels they are unable to ... ? O_o But I've got the point, will fix.

Reply:
As a single pronoun:
"they is unable to"
I'm not insisting but just in case there are any SJW folks on the horizon.

Reply:
I totally understand that, just didn't know that ["they" could be used in singular form](https://en.wikipedia.org/wiki/Singular_they)